plugin/phpactor.vim

```diff
@@ -708,38 +635,53 @@ function! phpactor#_rpc_dispatch(actionName, parameters)
     throw "Do not know how to handle action '" . a:actionName . "'"
 endfunction
 
-function! phpactor#_rpc_dispatch_input(type, parameters)
-    " Remove any existing output in the message window
-    execute ':redraw'
+function! phpactor#_rpc_dispatch_input_handler(Next, parameters, parameterName, result)
```

Comment:
Should `Next` be lower case?

Reply:
If I remember correctly yes.
A public function should start with a letter in upper case.
Exception for autoloaded function where everything is lower case (most of the time).
But I'm not 100% sure.

I will look into the error you mentioned.